sympy/core/operations.py

```diff
@@ -21,6 +25,19 @@ class AssocOp(Basic):
 
     This is an abstract base class, concrete derived classes must define
     the attribute `identity`.
+
+    Parameters
+    ==========
+
+    *args : Arguments which are operated
```

Comment:
I think the thing after the colon (if present) has to be a type.
```suggestion
    *args :
        Arguments which are operated
```